crypto/asn1/asn_mime.c

```diff
@@ -9,6 +9,7 @@
 
 #include <stdio.h>
 #include <ctype.h>
+#include "internal/ctype.h"
```

Comment:
In previous version it took internal/ctype.h alone, and I kind of appreciated it. Because `if (ossl_isupper) ossl_tolower` looks more consistent than `if (ossl_isupper) tolower`. But all right...